app/models/folio/bib_data.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Folio
+  # Bibliographic data from Folio.
+  class BibData
+    def initialize(request, _ = nil)
+      @request = request
+    end
+
+    attr_reader :request
+
+    def title
+      json.fetch('indexTitle', '')
```

Comment:
Is there a reason to pick `indexTitle` over `title`? The docs suggest indexTitle is intended for browse, not general display:

```

    "indexTitle": {
      "type": "string",
      "description": "Title normalized for browsing and searching; based on the title with articles removed"
    },
```

Reply:
The title field also displays the authors. For example: "Coffee / Nebiyu Assefa and Joanna Brown" whereas the `indexTitle` is the actual item title (e.g. "Coffee").

Reply:
But `indexTitle` has some other significant side-effects; for `a14439363`, the leading article gets dropped and proper nouns get down-cased:
> "title": "A history of Persia : from the beginning of the nineteenth century to the year 1858 / Robert Grant Watson ; with an introduction by Ali Ansari",

vs 

> "indexTitle": "History of persia : from the beginning of the nineteenth century to the year 1858",

Reply:
Switched this back to `title`. I'm splitting on `' / '` to remove the author part now.